.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules
```

Comment:
I think that this `/` is not needed here

Reply:
It also works